<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Registration Tool</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>851</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtImg1">
       <property name="text">
        <string>/Users/mehrdadnekopour/Downloads/CHEST 1.png</string>
       </property>
       <property name="placeholderText">
        <string>Enter Image Address</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnPreviewImg2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnPreviewImg1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btnFeatureBased">
       <property name="text">
        <string>Align Images (Feature-based Registration)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Image To Align</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="btnIntensityBased">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Align Images (Intesity-based Registration - Translation)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtImg2">
       <property name="text">
        <string>/Users/mehrdadnekopour/Downloads/CHEST 2.png</string>
       </property>
       <property name="placeholderText">
        <string>Enter Image Address</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Reference Image</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="btnRegister">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
